src/effects.rs

```diff
@@ -7,10 +7,12 @@ use crate::service::storage_service::{StorageRequest, StorageRequestPayload};
 use crate::service::{Service, StorageService};
 use crate::State;
 
+use crate::peer::binary_message::read::effects::effects as peer_binary_message_read_effects;
```

Comment:
We discussed that on call but can you please rename methods and filenames to their full size? `peer_binary_message_read_effects`.

Also in the bottom level, please reexport all the files like it's done here: https://github.com/tezedge/tezedge_redux/blob/master/src/peer/connection/outgoing/mod.rs

When there is a submodule, we just define a module: https://github.com/tezedge/tezedge_redux/blob/master/src/peer/connection/mod.rs